data.sql

```diff
@@ -8,4 +8,27 @@ INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) val
 INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) values  ('Angemon', '2005-06-12', 1, FALSE, -45);
 INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) values  ('Boarmon', '2005-06-07', 7, TRUE, 20.4);
 INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) values  ('Blossom', '1998-10-13', 3, TRUE, 17);
-INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) values  ('Ditto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+INSERT INTO animals(name, date_of_birth, escape_attemps,neutered, weight_kg) values  ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+/*Inserting into owners table*/
+INSERT INTO owners(id, full_name,age) VALUES (1, 'Sam Smith',34)
+INSERT INTO owners(id, full_name,age) VALUES (2, 'Jennifer Orwell',19)
+INSERT INTO owners(id, full_name,age) VALUES (3, 'Bob',45)
+INSERT INTO owners(id, full_name,age) VALUES (4, 'Melody Pond',77)
+INSERT INTO owners(id, full_name,age) VALUES (5, 'Dean Winchester',14)
+INSERT INTO owners(id, full_name,age) VALUES (6, 'Jodie Whittaker',38)
+
+/*Inserting into species*/
+INSERT INTO species(id,name) VALUES (1, 'Pokemon')
+INSERT INTO species(id,name) VALUES (2, 'Digimon')
+
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') where name = 'Agumon';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') where name = 'Gabumon' or name='Pikachu';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') where name = 'Devimon' or name='Plantmon';
```

Comment:
- [ ]  as per project requirement, `Bob owns Devimon and Plantmon`. 
Let's kindly make sure we fix that, to prevent any further errors later on when querying
![image](https://user-images.githubusercontent.com/87689547/165915244-51a80ff4-281e-4d75-a089-f78dfdee5f47.png)